src/pkg/k8s/images.go

```diff
@@ -68,6 +69,13 @@ findImages:
 			return nil, fmt.Errorf("unable to get the node %s", pod.Spec.NodeName)
 		}
 
+		requestedCPU := resource.MustParse(".5")
+		requestedMemory := resource.MustParse("64MiB")
+		if nodeDetails.Status.Allocatable.Cpu().Cmp(requestedCPU) >= 0 &&
+		   nodeDetails.Status.Allocatable.Memory().Cmp(requestedMemory) >= 0 {
```

Comment:
```suggestion
		if nodeDetails.Status.Allocatable.Cpu().Cmp(requestedCPU) < 0 ||
		   nodeDetails.Status.Allocatable.Memory().Cmp(requestedMemory) < 0 {
```
The logic here seems inverted.

Also, I think it is ok if we accept a node if its allocatable resources are equal, there is a chance it could fill up in the time we have remaining (there always could be a chance of that) but it would likely be better to try if we can than fail when things would otherwise have worked.

Reply:
Thanks for catching that!

Okay thanks for clarifying, that makes sense to me.